examples/remote_execution/buildbarn/README.md

```diff
@@ -1,30 +1,51 @@
-## Remote execution integration with Build Barn
+## Remote execution integration with Buildbarn
 
-This project provides a small example of what a project that utilizes [Build Barn](https://github.com/buildbarn) for RE might look like.
+This project provides a small example of what a project that utilizes [Buildbarn](https://github.com/buildbarn).
 
 In this document, we will go over the key configs used in this setup.
+Using a local docker-compose deployment from the [example deployment repo](https://github.com/buildbarn/bb-deployments)
```

Comment:
We may want some connective tissue here, to say that with a ready Buildbarn deployment the deploy section can be skipped. But for those who want to test remote execution, I think it's good to have instructions for that.
We can make it more succinct after we fix the instance-name prefix issue. But I would like to help people avoid that speed bump.

Reply:
That landed, can you add it?

Reply:
Yes!